airflow/www/templates/airflow/main.html

```diff
@@ -99,7 +99,7 @@
   {% if auth_manager.is_logged_in() %}
     {% call show_message(category='info', dismissible=true) %}
       <span>We have a new UI for Airflow 3.0</span>
-      <a href="{{ url_for('ui.index') }}">Check it out now!</a>
+      <a href="http://localhost:29091/webapp">Check it out now!</a>
```

Comment:
Hardcoded for now. This will get removed soon for airflow 3 and addding that through a proper config seems overkill in that context.